roles/kubernetes-apps/heapster/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Heapster | Lay Down Heaspter Templates
+  template: src={{item.file}} dest={{kube_config_dir}}/{{item.file}}
+  with_items:
+    - {file: grafana-deployment.yaml, type: deployment, name: monitoring-grafana}
```

Comment:
stylistically, we've been naming all resources ".yml" and not ".yaml". It's better to be consistent with current approach